migrations/20220321220618-rename-virtual-card-epire-date-to-expiry-date.js

```diff
@@ -0,0 +1,93 @@
+'use strict';
+
+import { omit } from 'lodash';
+
+import { crypto } from '../server/lib/encryption';
+
+module.exports = {
+  async up(queryInterface) {
+    const [virtualCards] = await queryInterface.sequelize.query(
+      `SELECT "id", "privateData" FROM "VirtualCards" WHERE "privateData" IS NOT NULL;`,
+    );
+    console.info(`Renaming 'expireDate' to 'expiryDate' for ${virtualCards.length} VirtualCards...`);
+    try {
+      await queryInterface.sequelize.transaction(async transaction => {
+        for (const virtualCard of virtualCards) {
+          const originalPrivateData = JSON.parse(crypto.decrypt(virtualCard.privateData));
+          if (!('expireDate' in originalPrivateData)) {
+            console.debug(`'expireDate' not in 'privateData' for VirtualCard with id=${virtualCard.id}`);
+            continue;
+          }
+          const privateData = JSON.stringify(
+            crypto.encrypt({
+              ...omit(originalPrivateData, ['expireDate']),
+              expiryDate: originalPrivateData.expireDate,
+            }),
+          );
+          console.debug(`Renaming 'expireDate' to 'expiryDate' for VirtualCard with id=${virtualCard.id}`);
+          await queryInterface.sequelize.query(
+            `
+              UPDATE "VirtualCards"
+              SET "privateData" = :privateData
+              WHERE "id" = :id;
+            `,
+            {
+              transaction,
+              replacements: {
+                privateData,
+                id: virtualCard.id,
+              },
+            },
+          );
+        }
+      });
+      console.info('Done.');
+    } catch (e) {
+      console.error('Oops, something went wrong and I rolled back the transaction.');
+      console.error(e);
+    }
+  },
+
+  async down(queryInterface) {
+    const [virtualCards] = await queryInterface.sequelize.query(
```

Comment:
:ice_cream: Since 90% of the code between up/down is shared, it could be nice to have a dedicated helper like `renameFieldInVirtualCardsPrivateData(oldName, newName)` in this file to simplify the code and prevent copy-paste oversights.

The thing could look like:

```es6
up: () => renameFieldInVirtualCardsPrivateData('expireDate', 'expiryDate');
down: () => renameFieldInVirtualCardsPrivateData('expiryDate', 'expireDate');
```